utils/common/src/all.records.ts

```diff
@@ -0,0 +1,78 @@
+import api, { Flatfile } from '@flatfile/api'
+import fetch from 'cross-fetch'
+
+const DEFAULT_PAGE_SIZE = 10_000
+
+export async function getRecordsRaw(
+  sheetId: string,
+  options: Flatfile.records.GetRecordsRequest = {}
+): Promise<Array<Flatfile.Record_>> {
+  const pageNumber = String(options.pageNumber ?? 1)
+  const pageSize = String(options.pageSize ?? DEFAULT_PAGE_SIZE)
+  // @ts-ignore
+  const query = new URLSearchParams({ ...options, pageNumber, pageSize })
```

Comment:
How do we get rid of this @ts-ignore... Maybe this?

```suggestion
  const query = new URLSearchParams({ ...options, pageNumber, pageSize }) as Record<string, any>
```